core/mgmt/v1beta/metric.proto

```diff
@@ -158,3 +158,36 @@ message ListPipelineTriggerChartRecordsResponse {
   // A list of pipeline trigger records.
   repeated PipelineTriggerChartRecord pipeline_trigger_chart_records = 1;
 }
+
+// CreditConsumptionChartRecord contains credit consumption metrics, aggregated
+// by owner and time frame.
+message CreditConsumptionChartRecord {
+  // Credit owner UUID.
+  string credit_owner_uid = 1;
```

Comment:
Hi @EiffelFly 
Do you think we should use `id` or `uid` here? Since most of our APIs use `id` in the path, I'm not sure if using `uid` will make it harder for the Console to integrate.

Reply:
I actually added this field in the record to have some sort of ownership as part of its data, but in fact the frontend shouldn't need to use it. I agree that in public endpoints the owner ID is more readable and straightforward, but in the request frontend can use `owner_name`.

Reply:
@donch1989 @jvallesm  I would suggest we use `id` like `users/summerbud` since that is the essence of resource oriented API design and console is quite happy with that since all the endpoint have unified behavior

Reply:
Got it.
I have a related question about this. If we don't include `owner_name` in the filter, what will it return?

Reply:
>  I would suggest we use id like users/summerbud

🤝  I guess exposing UIDs unnecessarily isn't great, I'll adjust the contract.

>  I have a related question about this. If we don't include owner_name in the filter, what will it return?

It'd aggregate credit over all owners. We don't have a use case for that and it is a resource-consuming operation so I'd make that filter mandatory. But, then, I guess it makes more sense as an explicit endpoint param. In the pipeline chart this is part of the filter but I think the same strategy applies.